circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -41,7 +39,7 @@
 class CuttingExperimentResults(NamedTuple):
     """Circuit cutting subexperiment results and sampling coefficients."""
 
-    quasi_dists: list[list[list[tuple[QuasiDistribution, int]]]]
+    results: SamplerResult | dict[str | int, SamplerResult]
```

Comment:
potentially interesting that the field name is plural (`results`) while the type (`SamplerResult`) is singular.  I wonder if it should just be `result`?  I've noticed similar inconsistencies, though, in other parts of the Qiskit ecosystem, such as the app modules iirc.

Reply:
Throughout our cutting workflows we've gone plural with `circuits` and `observables`, since they can be represented as a single instance or a dictionary of instances. My thought was that this applies exactly the same here.

Reply:
Yes, I agree with your point now.